conf/munin-docker.config

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Could you drop the docker/singularity specific configurations and instead use a more generalized approach? 

Have a single `base.config` that defines resource requests ( cpu, time, memory etc pp) but hardware configuration for munin stays in the munin.config on nf-core/configs. 

Then use this approach here:

https://github.com/nf-core/eager/blob/master/nextflow.config#L43-L62 

https://github.com/nf-core/eager/tree/master/conf

(not even having much more than just base, test, awsbatch and igenomes config here anymore - everything else is just a combination of default configs for specific clusters and your `base.config`) 

`nextflow run nf-core/rnafusion -profile munin,docker` should then be fine already 👍

Reply:
Done. Thanks for the help on Slack 🥇